README.md

```diff
@@ -124,6 +124,54 @@ The delete operation makes use of a single property ```tag``` and operates as de
 
 Note: Delete searches **all** repositories for tagged assets.
 
+# Deferred Deployment
+
+There is sometimes a desire to stage the deployment separately from performing the upload. This capability exists via
+deferred deployment for release artifacts (Snapshots are not supported). 
+
+The pom file requires configuration as described in [Example configuration](#example-configuration) with the addition of
+a ```stageLocally``` property, that can be present within the pom or on the command-line as seen in the example below.
+
+There is also the ability to use the ```altStagingDirectory``` property to specify the directory that the local staging
+will be performed within. If not specified, this will default to ```target/nexus-staging``` (and as such will be removed
+with ```mvn clean```).
```

Comment:
```suggestion
You can also use the ```altStagingDirectory``` property to specify the directory in which to perform the local staging. If not specified, this will default to ```target/nexus-staging``` and will be removed
with ```mvn clean```.
```